cms/wp/db/backup/import.php

```diff
@@ -0,0 +1,20 @@
+<?php
+/**
+ * Wordpress Database Backup Import.
+ *
+ * @file
+ * @package wp-deployer
+ */
+
+namespace Deployer;
+
+task(
+	'wp:db:backup:import',
+	function () {
+		$latestBackup = within( '{{backups}}', "find tmp -type f -printf '%T@ %p\n' | sort -n | tail -1 | cut -f2- -d\" \"" );
+		run( "cd {{app}} && zcat ../$latestBackup | {{wp}} db import " );
```

Comment:
This assumes that `$latestBackup` is always located (relatively) one level above `{{app}}`, but what if `{{backups}}` were set to something completely different like `/var/backups/`?